index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>판다마켓</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header>
+    <div class="gnb wrap">
+      <a href="/" class="logo">
+        <img src="./images/Property 1=sm.png" alt="판다마켓 로고">
+      </a>
+      <a href="/login" class="btn-login Text-lg semibold">로그인</a>
+    </div>
+  </header>
+
+  <main>
+    <section class="home-top-section">
+      <div class="home-top wrap">
+        <div class="home-top-content">
+          <p class="text-4xl bold">
+            일상의 모든 물건을 <br>
+            거래해 보세요
+          </p>
+          <a href="/items" class="btn-large text-xl semibold">구경하러 가기</a>
+        </div>
+        <img src="./images/Img_home_top.png" alt="판다 이미지">
+      </div>
+    </section>
+
+    <section class="home-section">
+      <div class="home">
+        <div class="home-content">
+          <img src="./images/Img_home_01.png" alt="인기 상품">
+          <div class="home-content-1">
+            <p class="keyword text-2lg bold">
+              Hot item
+            </p>
+            <p class="text-4xl bold">
+              인기 상품을<br>
+              확인해 보세요
+            </p>
+            <p class="home-describe text-2xl medium">
+              가장 HOT한 중고거래 물품을<br>
+              판다 마켓에서 확인해 보세요
+            </p>
+          </div>
+        </div>
+      </div>
+      <div class="home">
+        <div class="home-content">
+          <div class="home-content-2">
+            <p class="keyword text-2lg bold">
+              Search
+            </p>
+            <p class="text-4xl bold">
+              구매를 원하는<br>
+              상품을 검색하세요
+            </p>
+            <p class="home-describe text-2xl medium">
+              구매하고 싶은 물품은 검색해서<br>
+              쉽게 찾아보세요
+            </p>
+          </div>
+          <img src="./images/Img_home_02.png" alt="상품 검색">
+        </div>
+      </div>
+      <div class="home">
+        <div class="home-content">
+          <img src="./images/Img_home_03.png" alt="상품 등록">
+          <div class="home-content-3">
+            <p class="keyword text-2lg bold">
+              Register
+            </p>
+            <p class="text-4xl bold">
+              판매를 원하는<br>
+              상품을 등록하세요
+            </p>
+            <p class="home-describe text-2xl medium">
+              어떤 물건이든 판매하고 싶은 상품을<br>
+              쉽게 등록하세요
+            </p>
+          </div>
+        </div>
+      </div>
+    </section> 
+
+    <section class="home-bottom-section">
+      <div class="home-bottom wrap">
+        <div class="home-bottom-content">
+          <p class="text-4xl bold">
+            믿을 수 있는 <br>
```

Comment:
❗️ 수정요청

지금 디자인에서 괜찮을 것 같지만 공백은 의도치 않은 동작을 발생시킬 수 있습니다~ 
필요한 공백이 아니면 가능하면 습관적처럼 없애는 것을 추천드려요~

```suggestion
            믿을 수 있는<br>
```